WcaOnRails/config/locales/en.yml

```diff
@@ -2165,4 +2166,4 @@ en:
       apply_to_host: Apply to host
     other_suggestions:
       title: Other suggestions?
-      body_html: 'Do you have other suggestions for Instagram or other social media content? <a href="https://www.worldcubeassociation.org/contact/website">Let us know</a>!'
\ No newline at end of file
+      body_html: 'Do you have other suggestions for Instagram or other social media content? <a href="https://www.worldcubeassociation.org/contact/website">Let us know</a>!'
```

Comment:
I'd expect most text editors to put a `\n` character at the end of each line, have you configured yours in a specific way?
Except for file format which don't support it (eg: binary files), people usually always have a newline character at end of line.
Most command line tools (cat, head, tail, more, wc, etc...) have their output kind of broken if you don't (for instance the shell prompt ends up on the same line as the last line of the output), and it's also how the posix standard defines a line (you can check this [SO question](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline) for even more details).

Reply:
I'm dumb, I read the diff the other way around -_-
But I don't understand how this shows up, the missing newline seems to have been fixed in master already...
Anyway, will squash and merge!